code/datums/components/attachment_handler.dm

```diff
@@ -133,15 +133,17 @@
 	if(!can_attach(attachment, user, attachment_data))
 		return FALSE
 
-	if(!CHECK_BITFIELD(attachment_data[FLAGS_ATTACH_FEATURES], ATTACH_NO_HANDS))
-		var/obj/item/in_hand = user.get_inactive_held_item()
-		if(in_hand != parent)
-			to_chat(user, span_warning("You have to hold [parent] to do that!"))
-			return FALSE
+
 
 	var/do_after_icon_type = BUSY_ICON_GENERIC
 	var/attach_delay = attachment_data[ATTACH_DELAY]
 
+	if(!CHECK_BITFIELD(attachment_data[FLAGS_ATTACH_FEATURES], ATTACH_NO_HANDS))
```

Comment:
I'm a little concerned about bugs that'll pop up by allowing this but it's probably Fine

Reply:
I am pretty certain this will cause bugs with attachments that do stuff on equip, which I assume is what QV is referring to.

Also I know it already used it but don't use the define, it's pointless and doesn't even make it easier to read/take less space.